README.md

```diff
@@ -18,6 +18,7 @@ Options:
 - `-h` shows help output
 - `-t` enables server test mode
 - `-v` enables verbose mode
+- `-i` enables insecure mode, skipping server certificate validation
```

Comment:
The `curl` tool uses `-k` for basically the same function. Do you think it might be worth having the two tools use the same letter for this?

Reply:
Good idea. I'll update.